crypto/fipsmodule/service_indicator/service_indicator.c

```diff
@@ -346,6 +346,54 @@ void HMAC_verify_service_indicator(const EVP_MD *evp_md) {
   }
 }
 
+void HKDF_expand_verify_service_indicator(const EVP_MD *evp_md) {
```

Comment:
Can we combine `HKDF_expand_verify_service_indicator` and `HKDF_extract_verify_service_indicator` into one function called `HKDF_verify_service_indicator`? It seems like they're checking against the same parameters.

Reply:
Certainly; I was originally thinking they'd end up testing different things (`extract()` doesn't know about your Info, `expand()` doesn't know about the IKM or Salt), but I couldn't find any specific requirements for sizes on any of them.